CONTRIBUTING.md

```diff
@@ -1,6 +1,32 @@
-# Web World Wind Design and Coding Guidelines
+# Contributing to Web WorldWind
 
-## General
+The following are a set of guidelines for contributing to the NASA Web WorldWind project. These are mostly guidelines, not
+rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table of Contents
+
+[Asking Questions](#asking-questions)
+
+[Design and Coding Guidelines](#design-and-coding-guidelines)
+
+[Contributing](#contributing)
+- [Reporting Bugs](#reporting-bugs)
+- [Suggesting New Features](#suggesting-new-features)
+- [Pull Requests](#pull-requests)
+
+## Asking Questions
```

Comment:
Have you considered the optional SUPPORT.md file described in GitHub's documentation? If so, what are your thoughts on using that file vs including the documentation here?
https://help.github.com/articles/helping-people-contribute-to-your-project/
https://help.github.com/articles/adding-support-resources-to-your-project/